src/demo/OceanObject.js

```diff
@@ -64,7 +64,9 @@ export const loadAllFishPartImages = () => {
 };
 
 export const initMobilenet = () => {
-  return mobilenetModule.load(1, 0.25).then(res => (mobilenet = res));
+  return mobilenetModule
+    .load({version: 1, modelUrl: 'models/mobilenet/model.json'})
```

Comment:
Have we thought about how this will work in Code Studio?
We could make this URL a parameter that we pass in at init time (looks like we are chained quite deep in init methods here, so it may be easier to put it in as a state variable)
Or is there a way to load this from a data URL so we can just bundle this in our package (like we do for images)?

Reply:
I guess the `.bin` files above (referenced from within the JSON) are actually harder to account for.

Reply:
Yeah I was leaning towards making this URL a parameter because I was having trouble bundling these files the way we do for png

Reply:
But yeah it will likely require a bit of refactor...

Reply:
It looks like we could solve this by implementing an `IOHandler` and passing that in.
https://www.tensorflow.org/js/guide/save_load#loading_models_with_iohandlers